src/app/components/Modal/ModalDetailsTable.js

```diff
@@ -11,12 +11,12 @@ function chooseStyleAndAnswer(round) {
 }
 
 function ModalDetailsTable(quiz, humanData, computerData) {
-  const [
+  const {
     humanAnswers,
     computerAnswers,
     rightAnswers,
     paths,
-  ] = prepareDataForDetailsTable(quiz, humanData, computerData);
+  } = prepareDataForDetailsTable(quiz, humanData, computerData);
```

Comment:
good job!

Reply:
I think it was a good idea to move the logic elsewhere. Indeed it was hard to understand mixed with renders.